src/components/_project/project-description.module.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../base/module';
+
+.ProjectDescription {
+  background-color: $color-dove;
+
+  .galleryWrap {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: 40px;
+    @media (min-width: $desktop) {
+      flex-direction: row;
+    }
+    img {
+      border: 10px solid $color-white;
+      @media (min-width: $desktop) {
+        height: 550px;
+      }
+    }
+  }
+  .Content {
+    max-width: 1200px;
```

Comment:
I'd love to see you use the built in variables we have here so this would be like...desktop? I would just refer to the variables and pick the closest.

Reply:
I always forget to use those outside of using them for media queries, good reminder thank you!